docs/user/metacat/source/install.rst

```diff
@@ -482,27 +482,27 @@ search engine. Unfortunately the Solr Debian packages that come with the Ubuntu
 system are obsoleted and you have to install the binary packages by yourself. This section 
 provides guidance on how to setup Solr to run in production on *nix platforms, such as Ubuntu.
 
-Metacat supports Solr 8.8.2 and newer versions. You may download the binary releases from:
+Metacat only supports ``Solr 8.*``, ``NOT 9.*``. You may download the binary releases from:
 
 https://lucene.apache.org/solr/downloads.html
```

Comment:
suggestion - could use the direct link to v8, since we don't support v9? https://solr.apache.org/downloads.html#solr-8112

Reply:
The problem using the direct link is that the link will be invalid if a newer version comes up. It only keeps the newest versions on that page.